lms/djangoapps/certificates/tests/test_views.py

```diff
@@ -37,6 +37,7 @@
     BadgeAssertionFactory,
 )
 from lms import urls
+from util import organizations_helpers as organizations_api
```

Comment:
nitpick: this is really an "api client", so should this be 'as organizations_api_client'?